Benqi-Agents/DelegationMonitor/src/agent.ts

```diff
@@ -0,0 +1,88 @@
+import {
```

Comment:
Some recommendations  about variables naming: 
- `BenqiContract` instead of `BenQiContract`
- `delegateChangedEvent` seems to be referring to a single element, while it's an array of logs. something like `delegateChangedEvents` or `delegateChangedLogs` would be better imo.